src/libPMacc/include/algorithms/math/floatMath/floatingPoint.tpp

```diff
@@ -60,6 +72,29 @@ struct Float2int_rd<float>
     }
 };
 
+template<>
+struct Float2int_rn<float>
+{
+    typedef int result;
+
+    HDINLINE result operator( )(float value)
+    {
+#if __CUDA_ARCH__
+        return ::__float2int_rn( value );
+#else
+        if(value < 0.0f)
+            return -(*this)(-value);
+        static const result epsilon = 1e-7f;
```

Comment:
alternatively, [std::numeric_limits<T>::epsilon()](http://en.cppreference.com/w/cpp/types/numeric_limits/epsilon) is the right choice.